### DISCUSSION TITLE
Start conversation when logic triggers

### DISCUSSION THREAD

**Discussion by pyaniz** [python-telegram-bot/python-telegram-bot#2313] [Q&A]
Hello! I am trying to begin a conversation based on an alert but I can't figure out how to do it.

Essentially I have something running in python that will trigger a `Bot([token].send_message(...)` to a _specific_ user when something specific happens, and I want to prompt that user with 2 options and handle trough the ConversationHandler.
The bit I am needing help with is:
How do I set the `entry_point` of the conversation handler to be the `send_message` command?
Thanks!

**Comment by harshil21** [comment]
Hello, 

The `entry_point` of a conversation handler only accepts a list of Handlers (see [docs](https://python-telegram-bot.readthedocs.io/en/stable/telegram.ext.conversationhandler.html)).

The entry point can handle the options directly.

The handler you'd want to use is `MessageHandler`(if you're using a `KeyboardButton` for options) or a `CallbackQueryHandler`(if you're using `InlineKeyboardButton` for options)

You can have a look at this [example ](https://github.com/python-telegram-bot/python-telegram-bot/blob/master/examples/conversationbot2.py)if you're still confused
